compatibility/build.gradle.kts

```diff
@@ -10,6 +10,7 @@ repositories {
 
 dependencies {
     implementation("com.apollographql.federation", "federation-graphql-java-support")
+    implementation("com.graphql-java", "graphql-java", "20.0")
```

Comment:
no need for this dep, instead add `gradle.properties` with `graphql-java.version = 20.0` to override spring version